src/components/Sidebar/index.tsx

```diff
@@ -5,21 +5,18 @@ import DesktopSidebar from "./DesktopSidebar"
 import MobileSidebar from "./MobileSidebar"
 import { tBTCFill } from "../../static/icons/tBTCFill"
 import { tBTCOutline } from "../../static/icons/tBTCOutline"
-import { featureFlags } from "../../constants"
 
 const Sidebar = () => {
   const { pathname } = useLocation()
   const navItems: NavItemDetail[] = useMemo(() => {
-    const navItems = []
-
-    if (featureFlags.TBTC_V2) {
-      navItems.push({
+    const navItems: NavItemDetail[] = [
+      {
         text: "tBTC",
         activeIcon: tBTCFill,
         passiveIcon: tBTCOutline,
         href: "/tBTC/how-it-works",
-      } as NavItemDetail)
-    }
+      },
+    ]
 
     return navItems
   }, [pathname])
```

Comment:
Array can be returned immediately, unnecessary to store it in variable. 
Moreover the `pathname` is no longer a dependency.
```suggestion 
 const navItems: NavItemDetail[] = useMemo(() => ([
  {
    text: "tBTC",
    activeIcon: tBTCFill,
    passiveIcon: tBTCOutline,
    href: "/tBTC/how-it-works",
  },
]), [])
```

Reply:
Good catch! And I think we don't need `useMemo` at all here to be honest. The array is not that costly to create, because there are not any complicated operations here anymore. What do you think? 470d1974e5232584eeac3eda087b54409e28312b

Reply:
This component is most likely to be removed with #15, besides that I agree